### DISCUSSION TITLE
Help with creating a login_password item via rest api

### DISCUSSION THREAD

**Discussion by elan-bsaltzman** [semaphoreui/semaphore#777] [Q&A]
can you provide an example of a curl command to post - create a login_password type key? I don't understand how to create this, I'm probably missing something here. Thanks for your help.

ubuntuuser@ansible-semaphore-oapi3-21-04-demo2:~$ curl -X POST --header 'Content-Type: application/json' --header 'Accept: application/json' --header 'Authorization: Bearer xxxxxxxxxxxxxxxxxxxxxxxx' -d '{"name": "PAT3","type": "login_password" ,"project_id": 1 }' http://localhost:3000/api/project/1/keys -v
Note: Unnecessary use of -X or --request, POST is already inferred.
*   Trying 127.0.0.1:3000...
* TCP_NODELAY set
* Connected to localhost (127.0.0.1) port 3000 (#0)
> POST /api/project/1/keys HTTP/1.1
> Host: localhost:3000
> User-Agent: curl/7.68.0
> Content-Type: application/json
> Accept: application/json
> Authorization: Bearer xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
> Content-Length: 59
>
* upload completely sent off: 59 out of 59 bytes
* Mark bundle as not supporting multiuse
< HTTP/1.1 400 Bad Request
< Content-Type: application/json
< Date: Fri, 29 Oct 2021 14:26:30 GMT
< Content-Length: 38
<
{"error":"password can not be empty"}

**Comment by fiftin** [comment]
Please try this:

` '{"name": "PAT3","type": "login_password" ,"project_id": 1, "login_password": "{\"login\": \"***\",\"password\": \"***\"}" }'`

**Comment by elan-bsaltzman** [comment]
Thanks, that worked great!

**Comment by ahoiroman** [comment]
Should be added to the api docs I guess?
